mappings/markdown/Zeek/v1.3.0/conn_log/README.md

```diff
@@ -0,0 +1,82 @@
+
+
+ ### Conditional mapping:
+Fields described here are subject to dynamic mappings contingent on a conditional evaluation of source data.
+| OCSF                          | Raw                              | Zeek Field Description                                                       | Evaluation Conditions                                                                                     |
+| ----------------------------- | -------------------------------- | ---------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+| `activity_id`                 | `conn_state` | Possible connection states. | If "SF" or "RSTO" or "RSTR" or "RSTRH" or "SH" or "SHR" then "2" (Close) <br>if "S0" then "4" (Fail), <br>if "REJ" then "5" (Refuse), <br>if "OTH" or "S1" or "S2" or "S3" then "6" (Traffic) <br>else "6" (Traffic) <br>Type is Integer. |
+| `type_uid`                    | `class_uid` <br>+ `activity_id` | N/A | Calculate with: (`class_uid` * 100) + `activity_id` <br>Type is long_t. |
+| `traffic.bytes`               | `resp_bytes` <br>+ `orig_bytes`  | The total number of payload bytes sent by both the originator and responder. | Sum of `resp_bytes` + `orig_bytes` <br>Type is long_t.       |
+| `traffic.packets`             | `orig_pkts` <br>+ `resp_pkts`    | The total number of packets sent by both the originator and responder.       | Sum of `orig_pkts` + `resp_pkts` <br>Type is long_t.       |
+| `observables[].value`         | `id.orig_h_name.vals`            | The set of names observed for a given originator address.                    | In a record where <br>`observables[].name` = "src_endpoint.hostname" <br>and `observables[].type_id` = "1" (Type is Integer) <br>and`observables[].reputation.provider` = `id.orig_h_name.src` <br>and`observables[].reputation.base_score` = "0.0" (Type is float_t) <br>and `observables[].reputation.score_id` = "0" (Type is Integer) |
+| `observables[].value`         | `id.resp_h_name.vals`            | The set of names observed for a given responder address.                     | In a record where <br>`observables[].name` = "dst_endpoint.hostname" <br>and `observables[].type_id` = "1" (Type is Integer) <br>and`observables[].reputation.provider` = `id.resp_h_name.src` <br>and`observables[].reputation.base_score` = "0.0" (Type is float_t) <br>and `observables[].reputation.score_id` = "0" (Type is Integer) |
+
+
+ ### Unmapped (proposed):
+
+| OCSF                        | Raw              | Zeek Field Description                                                                  |
+| --------------------------- | ---------------- | --------------------------------------------------------------------------------------- |
+| `(bytes_missed)`            | `missed_bytes`   | Indicates the number of bytes missed in content gaps.                                   |
```

Comment:
We might want to add something to represent this

Reply:
Maybe we can track it via Issues?